contracts/contracts/VerificationGateway.sol

```diff
@@ -89,30 +89,31 @@ contract VerificationGateway
     @param hash BLS public key hash used as salt for create2
     @return BLSWallet at calculated address (if code exists), otherwise zero address
      */
-    function walletFromHash(bytes32 hash) public view returns (IWallet) {
+    // function walletFromHash(bytes32 hash) public view returns (IWallet) {
         //return wallet of hash registered explicitly
-        if (externalWalletsFromHash[hash] != IWallet(address(0))) {
-            return externalWalletsFromHash[hash];
-        }
-
-        address walletAddress = address(uint160(uint(keccak256(abi.encodePacked(
-            bytes1(0xff),
-            address(this),
-            hash,
-            keccak256(abi.encodePacked(
-                type(TransparentUpgradeableProxy).creationCode,
-                abi.encode(
-                    address(blsWalletLogic),
-                    address(walletProxyAdmin),
-                    getInitializeData()
-                )
-            ))
-        )))));
-        if (!hasCode(walletAddress)) {
-            walletAddress = address(0);
-        }
-        return IWallet(payable(walletAddress));
-    }
+        // if (externalWalletsFromHash[hash] != IWallet(address(0))) {
+        //     return externalWalletsFromHash[hash];
+        // }
+
+        // address walletAddress = address(uint160(uint(keccak256(abi.encodePacked(
+        //     bytes1(0xff),
+        //     address(this),
+        //     hash,
+        //     keccak256(abi.encodePacked(
+        //         type(TransparentUpgradeableProxy).creationCode,
+        //         abi.encode(
+        //             address(blsWalletLogic),
+        //             address(walletProxyAdmin),
+        //             getInitializeData()
+        //         )
+        //     ))
+        // )))));
+    //     IWallet walletAddress = registeredWalletFromHash[hash];
+    //     if (!hasCode(address(walletAddress))) {
+    //         walletAddress = IWallet(address(0));
+    //     }
+    //     return walletAddress;
+    // }
```

Comment:
1. Is there a reason to keep this code around? If so, might be better to move it into a docstring, other documentation, or a Solidity lib.
2. This will break [BlsWalletWrapper.Nonce](https://github.com/web3well/bls-wallet/blob/main/contracts/clients/src/BlsWalletWrapper.ts#L144), so we will need to update the client lib, publish a new version, and update consumers.

Reply:
1. No, this is a preliminary commit
2. I don't think this should break anything since the deleted function interface `walletFromHash` is the same is the hash/key mapping, now of the same name. All smart contract tests pass unchanged

Reply:
@kautukkundan as well as the specific tests for the fix, feel free to remove this too.